packages/itwin/property-grid/src/components/MultiElementPropertyGrid.tsx

```diff
@@ -112,41 +105,34 @@ export const MultiElementPropertyGrid = (props: PropertyGridProps) => {
       disableUnifiedSelection={true}
       onBackButton={() => {
         setContent(MultiElementPropertyContent.ElementList);
-        setAnimationForward(false);
       }}
       key={"SingleElementPropertyGrid"}
     />
   );
 
+  // since css animation requires all react components be rendered - just move non-active components to the side
+  const getAnimationStyle = (idx: MultiElementPropertyContent.PropertyGrid) => {
+    let style: React.CSSProperties;
+
+    if (idx === content) {
```

Comment:
Do we need this check?

Reply:
Just tested it, we don't need the check if we initialize style to an empty object. Having an empty object for style has the same visual effect as translate(0,0). I'll push a new commit